tests/test_minor_model.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+
+from models import Gene, Sample, Phenotype
+
+# TODO method for creating samples faster
+
+def test_gene_init():
+    gene = Gene('BAD')
+    assert gene.name == 'BAD'
+
+
+def test_sample_init():
+    genes = {Gene('BAD'): 1.2345, Gene('FUCA2'): 6.5432}
+
+    sample = Sample('Tumour_1', genes)
+
+    assert sample.name == 'Tumour_1'
+    assert all(isinstance(k, Gene) for k in sample.data.keys())
+    assert sample.data == genes
+
+
+def test_sample_from_names():
+    data = {'BAD': 1.2345, 'FUCA2': 6.5432}
+
+    sample = Sample.from_names('Tumour_1', data)
+
+    assert sample.name == 'Tumour_1'
+    assert all(isinstance(k, Gene) for k in sample.data.keys())
+    assert [k.name for k in sample.data.keys()] == ['BAD', 'FUCA2']
+    assert all(isinstance(v, float) for v in sample.data.values())
+    assert sample.data.values() == [1.2345, 6.5432]
```

Comment:
Just a small thing I found out recently: in Python 3.6 this line fails because `dict.values()` does not return a list but a custom `dict_values()` type. In older Python versions this could fail randomly if `sample.data` was a dict (there is no guaranteed order of items in dict prior to Python 3.6/3.7).

I will fix this soon in my PR.